tests/rates/testnetMarketMaker.spec.ts

```diff
@@ -56,14 +56,14 @@ describe("Test the TestnetMarketMaker module", () => {
       await createMockBalances(100, 1000)
     );
 
-    const trades = await marketMaker.prepareTradesToPublishForAsset(
+    const offers = await marketMaker.prepareOffersToPublishForAsset(
       buyAsset,
       sellAsset
     );
-    expect(trades).toBeDefined();
+    expect(offers).toBeDefined();
 
-    expect(trades.buy).toEqual(trade.buy); // Based on the publish fraction
-    expect(trades.sell).toEqual(trade.sell);
+    expect(offers.buy).toEqual(offer.buy); // Based on the publish fraction
```

Comment:
Had to look twice :D maybe you can rename either `offer` or `offers` if you have touch this again.

Reply:
Yep already done in #100